packages/perseus/src/widgets/image.tsx

```diff
@@ -273,25 +269,6 @@ class ImageWidget extends React.Component<Props> {
     }
 }
 
-const styles = StyleSheet.create({
```

Comment:
Just as a learning opportunity for me - I thought we were trying to move away from Less and towards scoped CSS-in-JS. Is there a reason why putting these styles in Less vs CSS-in-JS is beneficial?